mod1/highRoler.java

```diff
@@ -0,0 +1,80 @@
+package mod1;
+
+import java.util.Random;
+import java.util.Scanner;
+
+public class highRoler {
+    public static void main(String[] args) {
+       // set randomizer
+       Random randomizer = new Random();
+
+       // set scanner
+       Scanner myScanner = new Scanner(System.in);
+
+       // set variables
+       int roll, die, die2, die3, die4, die5, die6, die7, die8, die9, die10, Amount;
+       String stringroll, stringAmount;
+
```

Comment:
This should be something like

`
// get the number of sides on the die from the user

// generate an random into from die sides + 1

// if the roll == 1 print out critical failure

// else if the roll == max  print out critical success

// else print out you rolled num
`